pkg/storage/storage.go

```diff
@@ -183,6 +183,11 @@ func GetPlatformStorage(listers *regopclient.Listers) (imageregistryv1.ImageRegi
 	case configapiv1.AlibabaCloudPlatformType:
 		cfg.OSS = &imageregistryv1.ImageRegistryConfigStorageAlibabaOSS{}
 		replicas = 2
+	case configapiv1.NutanixPlatformType:
+		cfg.PVC = &imageregistryv1.ImageRegistryConfigStoragePVC{
```

Comment:
We don't recommend to use PVC on new platforms. It's better to have the image registry removed by default (like we have for bare metal) or to use object storage.

Reply:
Thanks for the review @dmage. Can you explain the rationale for the recommendation?